scripts/nouns/zkvote_contract.ts

```diff
@@ -17,10 +17,9 @@ export async function deploy(
   max_total_voting_weight: bigint,
 ): Promise<ZKVote> {
 
-  const verifier_contracts: Contract[] = await Promise.all([
-    (new NvoteVerifier__factory(deployer)).deploy(),
-    (new TallyVerifier__factory(deployer)).deploy(),
-  ]);
+  const verifier_contracts: Contract[] = [];
```

Comment:
Curious why the switch to serial deployment?

Reply:
Because I was getting a `Nonce too low error` when trying to deploy at the same time.

Reply:
Ah, right.  That makes sense.